### DISCUSSION TITLE
Unable to access internet on namespace with faucet

### DISCUSSION THREAD

**Discussion by shameem2001** [faucetsdn/faucet#4499] [Q&A]
Refer issue: https://github.com/faucetsdn/faucet/issues/4494

I have setup an OVS bridge with eth0 interface added as a port and namespace added as a port. I can do all qos and block operation with OVS commands in it. But when I set faucet controller to this setup. New flow rules are added and I can't no longer access the internet. Please give me a solution for this.

My OVS bridge setup is as follows:
OFPT_FEATURES_REPLY (OF1.3) (xid=0x2): dpid:0000b42e996236c9
n_tables:254, n_buffers:0
capabilities: FLOW_STATS TABLE_STATS PORT_STATS GROUP_STATS QUEUE_STATS
OFPST_PORT_DESC reply (OF1.3) (xid=0x3):
1(enp4s0): addr:b4:2e:99:62:36:c9
config: 0
state: LIVE
current: 1GB-FD COPPER AUTO_NEG
advertised: 10MB-HD 10MB-FD 100MB-HD 100MB-FD 1GB-FD COPPER AUTO_NEG AUTO_PAUSE AUTO_PAUSE_ASYM
supported: 10MB-HD 10MB-FD 100MB-HD 100MB-FD 1GB-FD COPPER AUTO_NEG AUTO_PAUSE AUTO_PAUSE_ASYM
speed: 1000 Mbps now, 1000 Mbps max
2(veth1): addr:2a:c6:3e:7e:66:fa
config: 0
state: LIVE
current: 10GB-FD COPPER
speed: 10000 Mbps now, 0 Mbps max
LOCAL(ovsbridge): addr:b4:2e:99:62:36:c9
config: 0
state: LIVE
speed: 0 Mbps now, 0 Mbps max
OFPT_GET_CONFIG_REPLY (OF1.3) (xid=0x9): frags=normal miss_send_len=0

I have access to internet if I don't set the controller. I have setup namespace on the OVS by following this link https://knowtoshare.wordpress.com/2016/04/30/open-vswitch-ovs-installation-and-configuration-part2/ . Here veth1 is for namespace.

But when I set up Faucet configuration. The setup breaks down, No traffic is allowed even if it is allowed in ACL.

Here is my Faucet configuration:

faucet.yaml
include:

/etc/faucet/acls/101.yaml
/etc/faucet/acls/102.yaml
dps:
ovs-br1:
dp_id: 0x0000b42e996236c9
interfaces:
1:
native_vlan: office
acls_in: [101]
2:
native_vlan: office
acls_in: [102]
vlans:
office:
vid: 100
description: "Office network 100 Vlan"

acls/101.yaml
acls:
101:
- rule:
actions:
allow: 1

acls/102.yaml
acls:
102:
- rule:
actions:
allow: 1

The demos only show ping between namespaces. It would be helpful to get this solution.

**Comment by muhammednihal05** [comment]
I also faced the same situation, any solution?
